rinad: make check broken test-routing e.nextHopAddresses

```
make[6]: Entering directory '/home/marc/sdk/rinad/build/src/ipcp/plugins/default'
clang++ -DHAVE_CONFIG_H -I. -I../../../../../src/ipcp/plugins/default -I../../../../src   -pthread -I/home/marc/sandbox/include -I/usr/include/libnl3  -I../../../../../src -I../../../../../src/common   -g -O2 -MT test_routing-test-routing.o -MD -MP -MF .deps/test_routing-test-routing.Tpo -c -o test_routing-test-routing.o `test -f 'test-routing.cc' || echo '../../../../../src/ipcp/plugins/default/'`test-routing.cc
../../../../../src/ipcp/plugins/default/test-routing.cc:876:4: error: no viable conversion from '_List_const_iterator<rina::NHopAltList>' to '_List_const_iterator<unsigned int>'
                        lit = e.nextHopAddresses.begin();
                        ^     ~~~~~~~~~~~~~~~~~~~~~~~~~~

```

--- a/rinad/src/ipcp/plugins/default/test-routing.cc
+++ b/rinad/src/ipcp/plugins/default/test-routing.cc
@@ -730,7 +730,7 @@ int getRoutingTable_MoreGraphEntries_True(bool lfa) {
 		bool ok = false;
 
 		std::cout << "Dest: " << e.address << ", Cost: " << e.cost <<
-				", NextHops: [";
+				", NextHopsAlts: {";
 
 		if (e.address < 1 || e.address > 7) {
 			std::cout << std::endl;
@@ -738,13 +738,20 @@ int getRoutingTable_MoreGraphEntries_True(bool lfa) {
 			break;
 		}
 
-		for (std::list<unsigned int>::const_iterator
-			lit = e.nextHopAddresses.begin();
-				lit != e.nextHopAddresses.end(); lit++) {
-			if (*lit == exp_nhops[e.address]) {
-				ok = true;
+		for (std::list<rina::NHopAltList>::const_iterator
+			altl = e.nextHopAddresses.begin();
+				altl != e.nextHopAddresses.end(); altl++) {
+
+			std::cout << "[";
+			for (std::list<unsigned int>::const_iterator
+				lit = altl->alts.begin();
+					lit != altl->alts.end(); lit++) {
+				std::cout << *lit << ", ";
+				if (*lit == exp_nhops[e.address]) {
+					ok = true;
+				}
 			}
-			std::cout << *lit << ", ";
+			std::cout << "] ";
 		}
 
 		if (!ok) {
@@ -753,7 +760,7 @@ int getRoutingTable_MoreGraphEntries_True(bool lfa) {
 			break;
 		}
 
-		std::cout << "]" << std::endl;
+		std::cout << "}" << std::endl;
 	}
 
 	if (lfa) {

